src/Repository/ProjectRepository.php

```diff
@@ -56,4 +56,14 @@ public function addConditionState(QueryBuilder $query, string $state): void
             ->andWhere('p.state = :state')
             ->setParameter('state', $state);
     }
+
+    /**
+     * Filter projects according to a Shuttle.
+     */
+    public function addConditionShuttle(QueryBuilder $query, string $shuttle): void
```

Comment:
use a type-hinted shuttle instead of string

```suggestion
    public function addConditionShuttle(QueryBuilder $query, Shuttle $shuttle): void
```